play.pokemonshowdown.com/src/battle-tooltips.ts

```diff
@@ -761,6 +761,19 @@ class BattleTooltips {
 			if (move.flags.wind) {
 				text += `<p class="movetag">&#x2713; Wind <small>(activates Wind Power and Wind Rider)</small></p>`;
 			}
+			// RBY healing move glitch
+			if (this.battle.gen === 1 && !toID(this.battle.tier).includes('stadium') &&
+				['recover', 'softboiled', 'rest'].includes(move.id)) {
+				const hpValues = [];
+				const hp = serverPokemon.maxhp - 255;
+				if (hp % 256 > 0) {
```

Comment:
```suggestion
				// glitches at HP values equal to `maxHP + 1 % 256` unless they are `0 % 256`
				const hp = serverPokemon.maxhp - 255;
				if (hp % 256 !== 0) {
```

Reply:
It's not guaranteed to be >0 here, so do you prefer `hp > 0 && hp % 256 !== 0`?

Reply:
On reflection, yes, I would. I think `%`'s behavior at negatives is really bad and unintuitive, and I'm unhappy it's what all modern languages use.